### PR TITLE
Email validation

### PR DESCRIPTION
**Pull request by RashidMenhas**
Hi 👋, In this PR I:
- Add email validation for upper case email.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -213,40 +102,41 @@ <h5 class="btn-connect">LET’S CONNECT</h5>
...
-                        <li>
+            <form id="form" action="https://formspree.io/f/xlevdjbz" method="post">
+                <ul>
+                    <li>
+                        <input class="name" type="text" maxlength="30" name="name" placeholder="Full Name" required>
+                    </li>
+                    <li>
+                        <input class="email" id="email" type="email" name="email" placeholder="Email Address" required>
+                    </li>
+                    <li id="error-msg"></li>
```
Comment (ichala) [inline index.html]: - [ ] Great Job the validations works perfectly , please note that the error message should be close to the submit button

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @RashidMenhas , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linters errors✅ 
- Validation Works✅ 

 ## Required Changes ♻️

- [ ] _**check the comments below**_


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (RashidMenhas) [reply index.html]: I applied the above mention issue.

**Review by emmiiorji — APPROVED** [review]
Hi @RashidMenhas , 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
congrats

## Good Highlights✔

- Requested changes have been implemented
- Linter checks are passing
- Good commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**